src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -7,45 +7,31 @@
 using Aspire.Dashboard.Otlp.Grpc;
 using Aspire.Dashboard.Otlp.Storage;
 using Aspire.Hosting.Utils;
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 using Microsoft.FluentUI.AspNetCore.Components;
 
 namespace Aspire.Dashboard;
 
-public class DashboardWebApplication : IHostedService
+public class DashboardWebApplication
 {
     private const string DashboardOtlpUrlVariableName = "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL";
     private const string DashboardOtlpUrlDefaultValue = "http://localhost:18889";
     private const string DashboardUrlVariableName = "ASPNETCORE_URLS";
     private const string DashboardUrlDefaultValue = "http://localhost:18888";
 
-    private readonly bool _isAllHttps;
     private readonly WebApplication _app;
-    private readonly ILogger<DashboardWebApplication> _logger;
 
-    public DashboardWebApplication(ILogger<DashboardWebApplication> logger)
+    public DashboardWebApplication()
     {
-        _logger = logger;
-
         var builder = WebApplication.CreateBuilder();
+
         builder.Logging.AddFilter("Microsoft.Hosting.Lifetime", LogLevel.None);
```

Comment:
We may not need these filters anymore since the dashboard isn't running in proc we don't need to worry about the noise it generates.